polus-czi-extract-plugin/README.md

```diff
@@ -0,0 +1,44 @@
+# Polus CZI Extraction Plugin
+
+This WIPP plugin will import individual fields of view from a CZI file (will only import one scene or collection of images). Images are exported as tiled tiffs using Bioformats. File names indicate their relative X and Y coordinates, as well as their channel, z-slice, and time-slice.
+
+The need for this plugin is due to the bioformats mechanism of importing CZI images. If the image in the CZI file is mosaiced (stitched based off image positions in Zen), then bioformats imports the mosaiced image. However, some people may want to preprocessing prior to mosaicing, or correlation based image stitching may want to be performed to improve the quality/accuracy of the stitching. Correlation based image stitching may be desired since small errors in the microscope stage may lead to minor offsets/defects in the mosaic based off of stage position alone.
```

Comment:
Lots of passive voice here. Something like: "However, certain use cases require preprocessing prior to mosaicing and alternative image stitching algorithms, such as correlation based, can improve the quality/accuracy of the stitching. Correlation based image stitching can address small errors in the microscope stage, which can lead to minor offsets/defects in the mosaic, better than stage position based mosaics."